sigmastate/src/main/scala/sigmastate/eval/CostingDataContext.scala

```diff
@@ -89,11 +89,9 @@ case class CAvlTree(treeData: AvlTreeData) extends AvlTree with WrapperOf[AvlTre
 
   override def wrappedValue: AvlTreeData = treeData
 
-  def startingDigest: Coll[Byte] = Colls.fromArray(treeData.digest)
+  override def keyLength: Int = treeData.keyLength
```

Comment:
Please provide ScalaDoc for all the public methods being touched in CAvlTree as well as for the class itself.

Reply:
done.